### ISSUE TITLE
Loss cannot converge

### ISSUE THREAD

**Issue by CYL0089** [luca-medeiros/lightning-sam#47]
![image](https://github.com/luca-medeiros/lightning-sam/assets/18739352/5e45de79-6765-49bb-8a2d-543d0452959c)
![image](https://github.com/luca-medeiros/lightning-sam/assets/18739352/f5ef0b66-d6e7-46be-9351-8208c13bc1e3)

As shown above，the fine-tune loss become larger, the config and training details are all follow yours. could you please tell me some possible reasons？

**Comment by CYL0089** [comment]
![image](https://github.com/luca-medeiros/lightning-sam/assets/18739352/62968d22-8241-4bc0-ae5d-6efef54ec4e0)

2 epoch, the losses have not decreased yet.

**Comment by SuroshAhmadZobair** [comment]
Hi @CYL0089 
have you tried reducing the batch_size number?

For me it converges on batch_size 2 but not on 3. I am using 2X3090s. 

Cheers
